plat/common/include/pci/pci_bus.h

```diff
@@ -244,11 +244,11 @@ static struct pci_bus_handler ph __unused;
 #define PCI_CONF_SUBSYS_ID_MASK     (0xFFFF)
 
 #define PCI_CONF_IRQ                (0X3C)
```

Comment:
????? What is this uppercase `X`???? Same for `PCI_CONF_IRQ_MASK`.... I know it is out of scope of this PR but I could not overlook this...